specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/examples/Settings/UpdateSetting_example.json

```diff
@@ -6,7 +6,7 @@
     "setting": {
       "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/settings/MCAS",
       "name": "MCAS",
-      "kind": "DataExportSetting",
+      "kind": "DataExportSettings",
       "type": "Microsoft.Security/settings",
```

Comment:
`type` is `readonly` property and cannot be sent in the request. pls remove it.

Reply:
Okay, thanks